src/twig/linearize/Callable.hpp

```diff
@@ -1,25 +1,32 @@
-template< typename Xstack, typename Ydata >
+template< typename Xstack, typename Ystack >
 class Callable {
   using Xdata = typename Xstack::value_type;
+  using Ydata = typename Ystack::value_type;
 
-  Xstack* stack;
+  Xstack* xStack_;
+  Ystack* yStack_;
   std::vector< Xdata > xBuffer{};
   std::vector< Ydata > yBuffer{};
 
   #include "twig/linearize/Callable/src/bin.hpp"
   #include "twig/linearize/Callable/src/grid.hpp"
 
 public:
-  Callable( Xstack& xStack ) : stack( &xStack ){}
-  void xStack( Xstack& stack ){ this->stack = &stack; }
+Callable( Xstack& xStack, Ystack& yStack ) : 
+  xStack_( &xStack ),
+  yStack_( &yStack )
+{}
+
+  void xStack( Xstack& stack ){ this->xStack_ = &stack; }
+  void yStack( Ystack& stack ){ this->yStack_ = &stack; }
```

Comment:
Are these functions actually used? This essentially would allow you to change the underlying stacks, and that may be a bad idea (what would happen if I only change the x stack?). You can just create a clean callable anyway.

Reply:
Currently there is no test that uses these functions. You would do this if you were going to re-linearize the same functor on a different grid. Seems like a bad idea anyway. I'll remove them.